utils/misc/misc.go

```diff
@@ -965,3 +967,35 @@ func GetInstanceID() string {
 	}
 	return ""
 }
+
+var jsonfast = jsoniter.ConfigCompatibleWithStandardLibrary
+
+// SanitizeJSON makes a json payload safe for writing into postgres.
+// 1. Removes any \u0000 string from the payload
+// ~2. Replaces any invalid utf8 characters using github.com/rudderlabs/rudder-go-kit/utf8~
+// 3. unmashals and marshals the payload to remove any extra keys
```

Comment:
> remove any extra keys

please elaborate

@lvrach do you remember why this is needed? It was introduced [here](https://github.com/rudderlabs/rudder-server/pull/4762)